developer/src/kmc-kmn/build.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+#
+# Compiles the kmc-kmn keyboard compiler.
+#
+
+# Exit on command failure and when using unset variables:
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+cd "$THIS_SCRIPT_PATH"
+
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+builder_describe "Build Keyman kmc kmn to kmx Compiler module" \
```

Comment:
Three three-letter file-extension-looking names in four tokens (all starting with `km`, no less!) makes for a pretty "fun" parse.  Nothing critical here, but I think this (or something similar) might be a tad more readable...

```suggestion
builder_describe "Build Keyman's kmc tool's kmn-to-kmx Compiler module" \
```

Reply:
Alternatively...

```suggestion
builder_describe "Builds Keyman Developer's kmn keyboard compiler module" \
```

would be a far better match for the package.json's name field.